MetaMask/metamask-extension
# Organizing storybook to echo app folder structure

Fixes: #12795 

Explanation: Organizing storybook to echo app folder structure inside of `ui/`

Updates include:
- UI components moved to `Components/UI`
- App components moved to `Components/App`
- Page components moved to `Pages/`
- Renaming all default/base stories to `DefaultStory` and to display as `Default`
- Removing spaces in the names to align with component names
- Updating all storybook components to start with a capital letter

This was originally in #12361 but moving to separate PR for easier review.

Manual testing steps:  
  - Go to latest CI build of storybook
<img width="921" alt="Screen Shot 2021-12-01 at 10 58 04 AM" src="https://user-images.githubusercontent.com/8112138/144296595-17ebf02a-f8ed-40b8-b6d9-63a8e8ba1db1.png">

  - See organized stories in sidebar.
  - In code check storybook title echos location of the component.
  
**Images**

<img width="1440" alt="Screen Shot 2021-11-22 at 5 08 16 PM" src="https://user-images.githubusercontent.com/8112138/142957534-9ca0dca6-7d9e-4810-a2be-de0d88a0081e.png">
